README.md

```diff
@@ -26,14 +27,14 @@ This package is a wrapper for FramePay offering out-of-the-box support for Redux
 For more information on FramePay see its [official documentation](https://rebilly.github.io/framepay-docs/) or [repository](https://github.com/Rebilly/framepay-docs).
 
 ### Demos      
-- [Use with react-redux](example/react-redux)    
-- [Use with react-router](example/react-router)   
-- [Use with ServerSide rendering](example/react-ssr)   
-- [Use with TypeScript](example/react-typescript)   
-- [Examples from README file](example/readme-example)
-- [Multiple Payment Methods](example/react-router/src/elements/MultiplePaymentMethods.js)      
-- [Separated fields](example/react-router/src/elements/PaymentCardSeparated.js)      
-- [Other Payment Methods](example/react-router/src/elements/OtherPaymentMethods.js)      
+- [Use with react-redux (codesandbox)](https://codesandbox.io/s/n089y731x4)    
+- [Use with react-router (codesandbox)](https://codesandbox.io/s/z2q2lx9ry4)      
+- [Examples from README file (codesandbox)](https://codesandbox.io/s/8ly7ml77y9)
+- [Multiple Payment Methods (codesandbox)](https://codesandbox.io/s/z2q2lx9ry4?module=/src/elements/MultiplePaymentMethods.js)      
+- [Multiple Payment Methods Short version (codesandbox)](https://codesandbox.io/s/z2q2lx9ry4?module=/src/elements/MultiplePaymentMethodsShort.js)      
```

Comment:
I get this error for example:

DependencyNotFoundError
Could not find dependency: 'framepay-react' relative to '/src/App.js'

Reply:
@adamaltman it doesn't work until the `framepay-react` doesn't released.
Right after the release i check and fix if necessary